nixpkgs_review/github.py

```diff
@@ -1,23 +1,35 @@
+import sys
 import json
+import os
+import urllib.error
 import urllib.parse
 import urllib.request
 from collections import defaultdict
 from typing import Any, DefaultDict, Dict, Optional, Set
 
+import backoff
+
+OWNER = "NixOS"
+REPO = "nixpkgs"
```

Comment:
This should use https://github.com/Mic92/nixpkgs-review/blob/12505d5348401b6c842cbe05701774fec9182cee/nixpkgs_review/cli/__init__.py#L185.